src/map/packets_struct.h

```diff
@@ -3319,6 +3319,20 @@ struct PACKET_ZC_MAKINGARROW_LIST {
 } __attribute__((packed));
 DEFINE_PACKET_HEADER(ZC_MAKINGARROW_LIST, 0x01ad);
 
+struct PACKET_ZC_SKILL_SELECT_REQUEST {
+	int16 packetType;
+	int16 packetLength;
+	int32 skillCount;
```

Comment:
this is not skill count this is special flag.
Flag same with flag from PACKET_CZ_SKILL_SELECT_RESPONSE if zero client may send additional packets
```
0 - LOWERVER_COMPATIBILITY
1 - SC_AUTOSHADOWSPELL
```

Reply:
hmm nice catch, I named it skillCount because we were writing `c` to it:
```C
WFIFOL(fd,4) = c;
```

and `c` here is the number of skills sent.

but you are right, checking a packet capture from official its value is 1. I will also fix the value sent